src/editor/EditorCtrl.ts

```diff
@@ -642,6 +647,7 @@ export default class EditorCtrl {
       $http.get(url)
             .then((response) => {
               $scope.draft = response.data;
+              checkIfPaidSectionShouldBeEnabled();
```

Comment:
I'd use here a functional approach, i.e not change the state in the function as the function name does not indicate it.

Can you refactor so it is the checkIfPaidSectionShouldBeEnabled does not amend the state and transition the state change to a more visible spot.

Reply:
It will make it easier to create tests for this once we transition to v2.